cloudinit/config/cc_phone_home.py

```diff
@@ -190,7 +191,7 @@ def handle(name, cfg, cloud, log, args):
         url_helper.read_file_or_url(
             url,
             data=real_submit_keys,
-            retries=tries,
+            retries=tries - 1,
```

Comment:
+1 on the logic correction here. While this does represent change in behavior, I'd defend that original logic was a bug. Generally this retry logic will only trigger in significant repeated error conditions.